fe/fe-core/src/main/java/org/apache/doris/common/Config.java

```diff
@@ -1010,6 +1010,18 @@
     @ConfField(mutable = true, masterOnly = true)
     public static int max_balancing_tablets = 100;
 
+    // Rebalancer type(ignore case): BeLoad, Partition. If type parse failed, use BeLoad as default.
+    @ConfField(masterOnly = true)
+    public static String rebalancer_type = "BeLoad";
+
+    // Valid only if use PartitionRebalancer.
+    @ConfField(mutable = true, masterOnly = true)
+    public static long rebalance_move_expire_after_access = 600; // 600s
```

Comment:
If only valid for PartitionRebalancer, the name would better add a partition prefix.

Reply:
ok